server/routes/__tests__/tags.spec.js

```diff
@@ -144,4 +133,101 @@ describe('GET /tags', () => {
       });
     });
   });
+
+  describe('/:id.json', () => {
+    describe('on error', () => {
+      beforeEach(() => {
+        jest.clearAllMocks();
+      });
```

Comment:
This is a catchall and probably could be on the parent _describe_ so it applies to all tests.